# zrj: add missing link against -lm in libHScryptonite-*.so
# NOTYPE  GLOBAL DEFAULT  UND log
# fixes www/hs-DAV (binutils 2.27 ld.gold is unhappy)
--- cryptonite.cabal.orig	2015-07-25 10:52:36.000000000 +0300
+++ cryptonite.cabal
@@ -72,6 +72,7 @@ Flag support_deepseq
   Manual:            True
 
 Library
+  extra-libraries: m
   Exposed-modules:   Crypto.Cipher.AES
                      Crypto.Cipher.Blowfish
                      Crypto.Cipher.Camellia
